test/org/sagebionetworks/bridge/dynamodb/DynamoBackfillDaoTest.java

```diff
@@ -13,34 +13,58 @@
 import org.joda.time.DateTime;
 import org.joda.time.DateTimeZone;
 import org.junit.After;
-import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.sagebionetworks.bridge.models.backfill.BackfillRecord;
 import org.sagebionetworks.bridge.models.backfill.BackfillStatus;
 import org.sagebionetworks.bridge.models.backfill.BackfillTask;
+
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
 
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBMapper;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBMapperConfig;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBMapperConfig.ConsistentReads;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBMapperConfig.SaveBehavior;
 import com.fasterxml.jackson.databind.JsonNode;
+import com.newrelic.agent.deps.com.google.common.collect.Lists;
```

Comment:
Remove NewRelic from our imports. Is there any way we can permanently disable this library from showing up in our code?